### PR TITLE
Wrapper tests 

### PR DESCRIPTION
**Pull request by Delfil**
Tests for all wrappers
Goals:
- 100% test coverage for all wrappers.
- Fix all found bugs

### CODE REVIEW

**Comment by Delfil** [comment]
All Wrapper s done. Ready for pull.

File: src/main/java/tudelft/ti2806/pl3/data/graph/DataNode.java [hunk 1]
```diff
@@ -163,7 +163,7 @@ public long getBasePairCount() {
 		return content.length;
 	}
 
-	public Set<Genome> getCurrentGenomeList() {
+	public Set<Genome> getCurrentGenomeSet() {
 		return currentGenomeList;
```
Comment (mathieupost) [inline src/main/java/tudelft/ti2806/pl3/data/graph/DataNode.java]: Rename the currentGenomeList variable too I guess?

File: src/main/java/tudelft/ti2806/pl3/data/wrapper/SingleWrapper.java [hunk 2]
```diff
@@ -12,7 +12,11 @@
 	private Wrapper target;
 	
 	public SingleWrapper(Wrapper target) {
```
Comment (TomBrouws) [inline src/main/java/tudelft/ti2806/pl3/data/wrapper/SingleWrapper.java]: Missing JavaDoc

**Comment by TomBrouws** [comment]
Looks good save for above comments.

**Comment by Delfil** [comment]
All done

**Comment by TomBrouws** [comment]
Looks good now!

**Pull request merged**